vsg/rules/generate/rule_017.py

```diff
@@ -0,0 +1,17 @@
+
+from vsg.rules import prefix_rule
+from vsg import utils
+
+
+class rule_017(prefix_rule):
+    '''
+    Generate rule 017 checks for prefixes in generate statement labels.
+    '''
+
+    def __init__(self):
+        prefix_rule.__init__(self, 'generate', '017', 'isGenerateLabel')
+        self.prefixes = ['gen_']
+        self.solution = 'Generate'
+
+    def _extract(self, oLine):
+        return utils.extract_class_identifier_list(oLine)
```

Comment:
Could you try the `utils.extract(label)` function?
It seems more appropriate in this instance.
This would apply to process_036 also.

Reply:
Sure, that makes sense.